cmd/service.go

```diff
@@ -138,12 +136,12 @@ func runServiceStartCmd(cmd *cobra.Command, args []string) {
 		}
 	}()
 
-	port := cmdconfig.DatabasePort()
+	port := viper.GetInt(constants.ArgPort)
```

Comment:
can we also remove the functions `cmdconfig.DatabasePort()` and `cmdconfig.ListenAddress()`